internal/server/config/config.go

```diff
@@ -342,15 +353,21 @@ func (cfg *Config) Verify() error {
 		}
 	}
 
-	if cfg.DispatchThrottling.Enabled {
-		if cfg.DispatchThrottling.Frequency <= 0 {
-			return errors.New("dispatchThrottling.frequency must be non-negative time duration")
+	// Tha validation ensures we are picking the right values for Check Dispatch Throttlign
```

Comment:
```suggestion
	// Tha validation ensures we are picking the right values for Check dispatch throttling
```
Typo in 'throttlign'

Reply:
This comment goes without saying though.. We don't need a godoc string telling us what the function name already tells us.  `VerifyCheckDispatchThrottlingConfig` is clear enough.